internal/primitive/transform/action/strings/split_with_intervals.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Linkall Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strings
+
+import (
+	"fmt"
+
+	"github.com/linkall-labs/vanus/internal/primitive/transform/action"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/arg"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/common"
+	"github.com/linkall-labs/vanus/internal/primitive/transform/context"
+)
+
+type splitWithIntervalsAction struct {
+	action.CommonAction
+}
+
+// NewSplitWithIntervalsAction["sourceJSONPath", "startPosition", "splitInterval", "targetJsonPath"].
+func NewSplitWithIntervalsAction() action.Action {
+	return &splitWithIntervalsAction{
+		CommonAction: action.CommonAction{
+			ActionName: "SPLIT_WITH_INTERVALS",
+			FixedArgs:  []arg.TypeList{arg.EventList, arg.All, arg.All, []arg.Type{arg.EventData}},
+		},
+	}
+}
+
+func (a *splitWithIntervalsAction) Init(args []arg.Arg) error {
+	a.TargetArg = args[3]
+	a.Args = args[:3]
+	a.ArgTypes = []common.Type{common.String, common.Number, common.Number}
```

Comment:
```suggestion
	a.ArgTypes = []common.Type{common.String, common.Int, common.Int}
```
we split type Number to Int and Float, pls update your local code from remote